src/design-system/organisms/TradeForm/TradeForm.tsx

```diff
@@ -382,7 +382,7 @@ export const TradeForm: React.FC<TradeFormProps> = ({
         .min(0.000001, `${t('minBuy')} 0.000001`)
```

Comment:
this may be outside of the scope of this ticket but i realize we don't have any max purchase validation based on total PMM holdings and it will allow the action to proceed to the wallet where it will fail